app/helpers/owners_helper.rb

```diff
@@ -0,0 +1,55 @@
+module OwnersHelper
+  def added_mailer_text(owner, user, rubygem)
+    if owner.id == user.id
+      I18n.t("mailer.owner_added.body_self", gem: rubygem.name)
+    else
+      I18n.t("mailer.owner_added.body_others", gem: rubygem.name, owner_handle: owner.handle)
+    end
+  end
+
+  def removed_mailer_text(owner, user, rubygem)
+    if owner.id == user.id
+      I18n.t("mailer.owner_removed.body_self", gem: rubygem.name)
+    else
+      I18n.t("mailer.owner_removed.body_others", gem: rubygem.name, owner_handle: owner.handle)
+    end
+  end
+
+  def owner_added_subject(owner, user, rubygem)
+    if owner.id == user.id
+      I18n.t("mailer.owner_added.subject_self", gem: rubygem.name)
+    else
+      I18n.t("mailer.owner_added.subject_others", gem: rubygem.name, owner_handle: owner.handle)
+    end
+  end
+
+  def owner_removed_subject(owner, user, rubygem)
+    if owner.id == user.id
+      I18n.t("mailer.owner_removed.subject_self", gem: rubygem.name)
+    else
+      I18n.t("mailer.owner_removed.subject_others", gem: rubygem.name, owner_handle: owner.handle)
+    end
+  end
+
+  def confirmation_status(ownership)
+    if ownership.confirmed?
+      content_tag(:span, class: "owners__icon") do
+        concat image_tag("/images/check.svg")
+        concat "Confirmed"
+      end
+    else
+      content_tag(:span, class: "owners__icon") do
+        concat image_tag("/images/clock.svg")
+        concat "Pending"
+      end
+    end
```

Comment:
Yes, can be done.
I understand that it is better to keep HTML things in views/ directory.
But the partial was getting a little ugly with the if else ladder of confirmation and MFA.
Can we make another partial for the rows maybe? @sonalkr132

Reply:
>  the partial was getting a little ugly with the if else ladder of confirmation and MFA

To clarify, I am not saying to remove the entire method. I am saying since `content_tag(:span, class: "owners__icon")` is there in both if and else blocks, may be we don't need to duplicate it or keep it in the helper method.